types/react-flatpickr/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for react-flatpickr 3.2
+// Project: https://github.com/coderhaoxin/react-flatpickr
+// Definitions by: begincalendar <https://github.com/begincalendar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Component } from 'react';
+import { EventCallback, Options } from 'flatpickr';
+
+interface DateTimePickerProps {
+    defaultValue?: string;
+    options?: Options;
+    onChange?: EventCallback;
+    value?: string;
+}
+
+declare class DatePicker extends Component<DateTimePickerProps, void> {}
```

Comment:
Hi Andy, thanks for reviewing this. 

I tried to make the changes you suggested, but I'm getting a linting error.
It appears [this rule](https://github.com/Microsoft/dtslint/blob/master/src/rules/strictExportDeclareModifiersRule.ts) (which you appear to be the author of), is the cause. It appears that rule has broken a lot of existing definitions, other than just this one that I'm trying to introduce.

Could you please provide some feedback on this issue (e.g. elaborate on how to conform to that new linting rule)?